### ISSUE TITLE
Pytorch , TF Lite  Model Loading and Logging

### ISSUE THREAD

**Issue by noshluk2** [HaiderAbasi/Raspberry-Pi-4-Computer-Vision-#6]
Setting up Tflite and Pytorch model running on RPI with easy to use on PC aswell

**Comment by HaiderAbasi** [comment]
@noshluk2 
**TFLite-> Looks like a no-show.
Why?
Speed:**

**1) Video example** 

![Image](https://user-images.githubusercontent.com/49869365/223345314-30f450a7-c064-4c84-91c8-3b28d9c28e90.png)[Video](https://www.youtube.com/watch?v=aimSGOAUI8Y&ab_channel=EdjeElectronics)

**2) Community** 

![Image](https://user-images.githubusercontent.com/49869365/223345739-27779493-3d1d-4931-95d5-d0e1cc1e5b79.png)[StackOverflow](https://stackoverflow.com/questions/67373721/raspberry-pi-4-very-slow-predictions-with-tensorflowlite-object-detection-api)


**3) YoloV4-tiny on NCNN (Q-Engineer)**

![Image](https://user-images.githubusercontent.com/49869365/223346041-a29aa820-d59b-4cdc-a482-89089e112dd1.png)
[YoloV4-tiny_NCNN](https://github.com/Qengineering/YoloV4-ncnn-Raspberry-Pi-4)

======================================================================

**Why is this a concern?**

No speed No tracking as most trackers work on tracking by detection. Meaning they run detection on every frame so slowness overpowered

**Why do we need tracking?**

**We need tracking to Know!**
- This is the same car that we saw one frame ago
- This is the same car we counted one frame ago (Car counter)
- This is the same car that was at point A then and Point B now (Overspeed detection)
- This is the same car whos identity we have verified . No need to do it again (Number plate recognition)

**Whats the solution?**

- Stick to Ncnn 
- Try to give inference speed of each of the top models (YoloX-Nano - NanoDet - SSD) and get viable prediciton resultss
- Get this working: https://github.com/Qengineering/Traffic-Counter-RPi_64-bit

**Comment by noshluk2** [comment]
I am shifting your comment to issue #20  . As this is for Tflite and Pytorch. This is an enhancement task. I will look into NCNN working on rpi and will update you on issue #20

**Comment by noshluk2** [comment]
### Model : [detect.tflite](https://github.com/Qengineering/TensorFlow_Lite-Tracking-RPi_64-bit/blob/main/detect.tflite)
### Video Input : [Walkers.mp4](https://github.com/Qengineering/TensorFlow_Lite-Tracking-RPi_64-bit/blob/main/Walkers.mp4)
### Result


![Image](https://user-images.githubusercontent.com/29896241/223352815-7afcb50b-d64c-4add-8abc-72fbc9f86d3b.png)

**Comment by HaiderAbasi** [comment]
@noshluk2 
Actually YoloV4 or any high accuracy model is a no show on tflite.
But if you got tflite running.

Can you test this [TfliteTrackerbyQengineering](https://github.com/Qengineering/TensorFlow_Lite-Tracking-RPi_64-bit)
 and what inference you get. For the moment. Let say we ignore the accuracy and there are 20 cars and car counter counts 40 and we are fine with it. 
taking this repo as an example. Try to see if you can get this running with .py script that you have and the rest of this running at the backend or converted to .py eitherway we want to say if it runs with a fair speed . (> 7 FPS)
